--- a/gasket_core.c
+++ b/gasket_core.c
@@ -1372,7 +1372,7 @@
 /* File operations for all Gasket devices. */
 static const struct file_operations gasket_file_ops = {
 	.owner = THIS_MODULE,
-	.llseek = no_llseek,
+	.llseek = noop_llseek,
 	.mmap = gasket_mmap,
 	.open = gasket_open,
 	.release = gasket_release,
@@ -1838,7 +1838,7 @@
 	internal = &g_descs[desc_idx];
 	mutex_init(&internal->mutex);
 	memset(internal->devs, 0, sizeof(struct gasket_dev *) * GASKET_DEV_MAX);
-	internal->class =
-		class_create(driver_desc->module, driver_desc->name);
+	internal->class =
+		class_create(driver_desc->name);
 
 	if (IS_ERR(internal->class)) {
--- a/gasket_interrupt.c
+++ b/gasket_interrupt.c
@@ -157,8 +157,8 @@
 	trace_gasket_interrupt_event(interrupt_data->name, interrupt_index);
 	read_lock(&interrupt_data->eventfd_ctx_lock);
 	ctx = interrupt_data->eventfd_ctxs[interrupt_index];
 	if (ctx)
-		eventfd_signal(ctx, 1);
+		eventfd_signal(ctx);
 	read_unlock(&interrupt_data->eventfd_ctx_lock);
 
 	++(interrupt_data->interrupt_counts[interrupt_index]);
